### PR TITLE
Add function for generating changelog

### PR DESCRIPTION
**Pull request by janmisar**
New function which prints all commit messages since last merge.

### CODE REVIEW

**Comment by beranm14** [comment]
Squash the commits, please

File: scripts/helper_functions.sh [hunk 1]
```diff
@@ -102,6 +102,14 @@ commits_count() {
   curl -s --HEAD --header "PRIVATE-TOKEN: $SECRET_GITLAB_ACCESS_TOKEN" "${CI_SERVER_URL}/api/v4/projects/$CI_PROJECT_ID/repository/commits?per_page=1&ref_name=$CI_COMMIT_REF_NAME" | grep x-total: | cut -d " " -f2
 }
 
+generate_changelog() {
+  # fetch all commits between current SHA and SHA-1 (previous commit on autodeploy branches should be merge commit so it returns list of changes since last merge == since last version)
+  curl -s -H "PRIVATE-TOKEN: $SECRET_GITLAB_ACCESS_TOKEN" "$CI_SERVER_URL/api/v4/projects/$CI_PROJECT_ID/repository/compare?from=${CI_COMMIT_SHA}~1&to=$CI_COMMIT_SHA" |
```
Comment (beranm14) [inline scripts/helper_functions.sh]: Is there any reasoning behind the usage of `${variable}` and `$variable`? Tbh, we had a discussion with Sandra while back that if the variable is part of a string, it should be `${variable}` for better readability. Could you please change it that way?

**Comment by janmisar** [comment]
> Squash the commits, please

image

Feel free..? 🤔

**Comment by beranm14** [comment]
Tbh, this squash options usually generate commit message as a list of all commit messages from the squashed commits, that's why I'd rather prefer to squash commits by myself and change commit message to something reasonable. In this case, I guess it could be "Add function for generating changelog".
What do u think?

**Comment by janmisar** [comment]
I don't use squash anywhere, so I have no idea how it works 😄 I thought this button is the only way to do it, but OK I will google that 🤓

**Comment by janmisar** [comment]
OK, that's all I can do 🤷‍♂️

**Comment by beranm14** [comment]
Looks great! Thanks :-)

**Pull request merged**